index.js

```diff
@@ -66,25 +66,44 @@ module.exports = function (PIXI)
                         sprite.dimension.w / resolution,
                         sprite.dimension.h / resolution
                     );
-                    var crop = new PIXI.Rectangle(
-                      0,
-                      0,
-                      sprite.dimension.w / resolution,
-                      sprite.dimension.h / resolution
-                    );
+
+                    var crop;
+                    if (Number(PIXI.VERSION.charAt(0)) >= 4) {
+                        crop = new PIXI.Rectangle(
+                            0,
+                            0,
+                            sprite.dimension.w / resolution,
+                            sprite.dimension.h / resolution
+                        );
+                    } else {
+                        crop = frame.clone();
```

Comment:
oh, I'm so sorry. I've just looked at your last commit and not the whole diff. I apologise for the confusion. Let me quickly test it and then I'll merge and release.